ui/src/app/views/login.html

```diff
@@ -1,5 +1,7 @@
 <div layout="column" layout-align="center center">
-    <img class="logo" ng-src="assets/images/logo.png">
+    <a href="https://github.com/scorelab/Bassa" target="_blank">
```

Comment:
Instead of having Link on main logo, can we create a complete footer at login page to introduce Bassa? @demon-36

Reply:
@vivonk I think the footer on login page will look weird, so If we don't want that link on login page then we can link the same with the Bassa logo on nav-bar on the main screen after login.

Reply:
Merged it for the main screen. Thanks @demon-36